### PR TITLE
Testimonials sections

### PR DESCRIPTION
**Pull request by Vadym-Mazurenko**
### Asana Task link:

- [Asana Task](https://app.asana.com/0/1206375175000843/1207965047617511/f)

### Demo links

- [Store](https://wuffes-store.myshopify.com/?preview_theme_id=130847998087)
- [Editor](https://admin.shopify.com/store/wuffes-store/themes/130847998087/editor?section=template--16238227357831__testimonials_full_width_kmk8m8&block=template--16238227357831__testimonials_full_width_kmk8m8__testimonials_slide_UGwK67)

### Checklist

- [x] Tested on mobile
- [x] Tested on desktop
- [x] Requested review from QA (if applicable)

- [x] Requested approval from 1 senior dev on Github and in Asana task
- [x] Moved task to "QA" status in Asana

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207991998419145

### CODE REVIEW

File: sections/testimonials.liquid [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+    endif
+  endif
+-%}
+
+{%- capture sizes -%}
+  {% if section.settings.full_width -%}
+    100vw
+  {%- else -%}
+    (min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 100 }}px, (min-width: 750px)
+    calc(100vw - 10rem), 100vw
```
Comment (alex-wuffes) [inline sections/testimonials.liquid:46]: @Vadym-Mazurenko This image [doesn't look](https://tppr.me/jTq7dr) 100vw to me on mobile.

File: sections/testimonials.liquid [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+      assign fix_ratio = true
+    endif
+  endif
+-%}
+
+{%- capture sizes -%}
+  {% if section.settings.full_width -%}
+    100vw
+  {%- else -%}
+    (min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 100 }}px, (min-width: 750px)
```
Comment (alex-wuffes) [inline sections/testimonials.liquid:45]: @Vadym-Mazurenko It's [not](https://tppr.me/bogUk6) 1100 px on desktop. Please, review this image sizes.

**Review by alex-wuffes — CHANGES_REQUESTED** [review]

File: assets/section-multicolumn.css [hunk 3]
```diff
@@ -29,7 +29,7 @@ slideshow-component {
   display: none;
 }
 
-.slider-counter.caption {
+.multicolumn-list .slider-counter.caption {
   display: none;
 }
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply assets/section-multicolumn.css:34]: fixed

File: assets/section-testimonials-full-width.css [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+  transform: translate(-50%) translateZ(0);
+  box-sizing: border-box;
+}
+
+.featured_story__slide_content_wrap {
+  min-height: 18rem;
+  padding: 3rem 3.6rem;
+}
+
+.slide_content__heading {
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply assets/section-testimonials-full-width.css]: fixed

File: assets/section-testimonials-full-width.css [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+}
+
+@media screen and (min-width: 460px) {
+}
+
+@media screen and (min-width: 500px) {
+}
+
+@media screen and (min-width: 680px) {
+}
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply assets/section-testimonials-full-width.css]: fixed

File: assets/testimonials.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  const oldPrevButton = document.querySelector('.slider-button--prev');
+
+  newNextButton.addEventListener('click', function () {
+    oldNextButton.click();
+  });
+
+  newPrevButton.addEventListener('click', function () {
+    oldPrevButton.click();
+  });
+});
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply assets/testimonials.js]: fixed

File: sections/testimonials-full-width.liquid [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+          "type": "inline_richtext",
+          "id": "description",
+          "default": "Slide description",
+          "label": "Slide description"
+        },
+        {
+          "type": "inline_richtext",
+          "id": "steps_per_day",
+          "default": "X steps / per day",
+          "label": "Staps per day"
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply sections/testimonials-full-width.liquid]: fixed

File: sections/testimonials.liquid [hunk 8]
```diff
@@ -0,0 +1,531 @@
...
+      "step": 4,
+      "unit": "px",
+      "label": "t:sections.all.padding.padding_bottom",
+      "default": 36
+    }
+  ],
+  "blocks": [
+    {
+      "type": "testimonials_video",
+      "name": "testimonials_video",
```
[the remark this answers is not in the document]
Comment (Vadym-Mazurenko) [reply sections/testimonials.liquid]: fixed

File: sections/testimonials.liquid (hunk 1 above)
Comment (Vadym-Mazurenko) [reply sections/testimonials.liquid]: fixed

File: sections/testimonials.liquid (hunk 2 above)
Comment (Vadym-Mazurenko) [reply sections/testimonials.liquid]: fixed

File: sections/testimonials-full-width.liquid [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+        },
+        {
+          "type": "testimonials_slide"
+        },
+        {
+          "type": "testimonials_slide"
+        }
+      ]
+    }
+  ]
```
[the remark this answers is not in the document]
Comment (alex-wuffes) [reply sections/testimonials-full-width.liquid]: @Vadym-Mazurenko That's better, but section 1 still looks off. 
image

I suggest adding a proper color schema into the preset

File: sections/testimonials.liquid [hunk 10]
```diff
@@ -0,0 +1,531 @@
...
+              >
+                <h3>{{ block.settings.item_title }}</h3>
+                <div class="block_video_wrapper">
+                  {%- if block.type == 'testimonials_video' -%}
+                    <deferred-media
+                      class="{%- if block.settings.video_autoplay -%}video_autoplay{% endif %} video-section__media deferred-media gradient global-media-settings{% if block.settings.full_width %} global-media-settings--full-width{% endif %}{% if fix_ratio %} media-fit-cover{% endif %}{% if block.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}"
+                      data-media-id="{{ video_id }} "
+                      {% if block.settings.video.preview_image != null %}
+                        style="--ratio-percent: {{ 1 | divided_by: block.settings.video.preview_image.aspect_ratio | times: 100 }}%;"
+                      {% endif %}
```
[the remark this answers is not in the document]
Comment (alex-wuffes) [reply sections/testimonials.liquid:124]: @Vadym-Mazurenko I can still [find](https://tppr.me/2ynvRi) blocks that don't have this attribute

**Review by alex-wuffes — CHANGES_REQUESTED** [review]
@Vadym-Mazurenko I left a few comments in some of the previous conversations and marked them unresolved

**Comment by alex-wuffes** [comment]
Duplicate https://github.com/Wuffes/wuffes-OS2.0/pull/13

**Pull request closed without merging**